docs/components/home/navbar.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable no-nested-ternary */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, { useState, useEffect } from 'react';
+import { useRouter } from 'next/router';
+import { Link, Box } from '@hack4impact-uiuc/bridge';
+
+const NavBar = () => {
+  const router = useRouter();
+  const [showNav, toggleNav] = useState(false);
+
+  const onToggle = () => {
+    toggleNav(!showNav);
+  };
+
+  const [windowDimensions, setWindowDimensions] = useState({ width: 1000, height: 1000 });
```

Comment:
Could this instead use the `useWindowDimensions` hook written?

Reply:
oh yeah I just removed `useWindowDimensions` because it doesn't work since its nextjs and server rendering